lib/src/base-styles/_nx-variables.scss

```diff
@@ -65,9 +65,14 @@
                             0 1px 2px 0 rgba(91, 95, 141, 0.2),
                             0 1px 1px 0 rgba(124, 128, 179, 0.2);
 
+  // NxDrawer
+  --nx-box-shadow-drawer: -34px 0 50px -32px hsla(0, 0%, 50%, 0.5);
+
   @include nx-dark-mode-helpers.dark-mode {
     --nx-border-grid: var(--nx-border-default);
     --nx-border-width: 2px;
+    // NxDrawer
+    --nx-box-shadow-drawer: -34px 0 50px -32px hsla(231, 41%, 10%, 0.75);
```

Comment:
We should probably add a comment explaining why the box shadow parameters are the way they are:

The mockups indicate that the shadow should be as if it is on an element that is shorter than the drawer such that its top and bottom are offset inwards by 32px each.  On this hypothetical element, the mockups indicate the shadow should have a spread of zero.  We can emulate that, without needing the extra element itself, by setting the actual spread to -32px, thus bringing in size of the shadow inwards by that amount on all sides.  But of course, we don't want it to be brought inward on the left, so we cancel that effect out on that side by adding -32px to the shadow's x-offset.  The mockups specify a further x-offset of -2px, resulting in a total of -34px.  On the right side of course, it doesn't matter how the shadow ends up as the element is against the edge of the viewport.

The y-offset can be zero as the shadow should be vertically centered relative to the drawer.  Finally, the blur comes directly from the mockup where it is specified as 50px.

Reply:
> We should probably add a comment explaining why the box shadow parameters are the way they are:

Great idea. Would you like me to use the full explanation here, or pare it down where possible?

Reply:
> A spread of -32px sets the spread inward to create the effect of a smaller, vertically centred element behind the drawer, as displayed in the mocks. The x-offset counteracts the shadow’s spread (to prevent it from being brought inward on the left), plus a further -2px of x-offset defined in the mocks. The Y-offset can be zero as the shadow should be vertically centred relative to the drawer. The blur comes directly from the mocks where it is specified as 50px.

Sounds good

Reply:
> > A spread of -32px sets the spread inward to create the effect of a smaller, vertically centred element behind the drawer, as displayed in the mocks. The x-offset counteracts the shadow’s spread (to prevent it from being brought inward on the left), plus a further -2px of x-offset defined in the mocks. The Y-offset can be zero as the shadow should be vertically centred relative to the drawer. The blur comes directly from the mocks where it is specified as 50px.
> 
> Sounds good

[1e6dd43](https://github.com/sonatype/sonatype-react-shared-components/pull/1097/commits/1e6dd43f403a4ffc331e913e7429bae247debfa8)